### DISCUSSION TITLE
Early stopping of process?

### DISCUSSION THREAD

**Discussion by sonoff2** [rapidfuzz/RapidFuzz#346] [Q&A]
Is there any way to do early stopping of process function: as soon as one elements reaches a score of 100 (or any other score), return the element? I was not able to find a solution to this (except writing my own process function)

**Comment by maxbachmann** [comment]
`procesa.extractOne` searches for the best match, so it exits early in case it encounters a perfect match (element with a score of 100).
There is no function searching for the first element with a score >= score_cutoff yet.
